compiler-core/src/type_.rs

```diff
@@ -877,6 +877,49 @@ impl ValueConstructorVariant {
             ValueConstructorVariant::Record { field_map, .. } => field_map.as_ref(),
         }
     }
+
+    /// If this is the constructor of a module function, returns the function's
+    /// module and name.
+    ///
+    fn function_module_and_name(&self) -> Option<(EcoString, EcoString)> {
+        match self {
+            ValueConstructorVariant::ModuleFn { name, module, .. } => {
+                Some((module.clone(), name.clone()))
+            }
+
+            ValueConstructorVariant::ModuleConstant { .. }
+            | ValueConstructorVariant::LocalConstant { .. }
+            | ValueConstructorVariant::LocalVariable { .. }
+            | ValueConstructorVariant::Record { .. } => None,
+        }
+    }
+
+    /// If this is a constructor for an argument, returns the name of the
```

Comment:
What's a constructor for an argument? Arguments don't have constructors, types and values have constructors.